### PR TITLE
Update zipping method to support java based extraction

### LINKED ISSUE #42: Java ZipInputStream cannot unzip update zip file
**Description:**

WUM update creator tool only inflates the file, not directories. This works fine for most of the unzipping tools.
But if we try to unzip via a java program it fails with "only DEFLATED entries can have EXT descriptor"

This is a limitation in ZipInputStream.

**Suggested Labels:**

Type/Bug

**Suggested Assignees:**

@irshadnilam

**Affected Product Version:**
1.1.0-m1

**OS, DB, other environment details and versions:**    
All

**Steps to reproduce:**
- Create a update zip file from update-creator-tool
- Unzip  the file using java using ZipInputStream

**Related Issues:**

N/A

### PR DESCRIPTION
**Pull request by irshadnilam**
## Purpose
With the current approach, it is not possible to unzip update zip packs created from update creator tool using Java util.zip library. This PR resolves #42 

## Goals
To support java based extraction to the update zips created from update creator tool.

## Approach
Remove /mholt/archiver based zipping. 
Use archive/zip based zipping.

## User stories
N/A

## Release note
N/A

## Documentation
N/A

## Training
N/A

## Certification
N/A

## Marketing
N/A

## Automation tests
N/A

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? N/A
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Samples
N/A

## Related PRs
N/A

## Migrations (if applicable)
N/A

## Test environment
Unzipping works with java version "1.8.0_131"

## Learning
N/A

### CODE REVIEW

**Comment by ChanakaCooray** [comment]
@irshadnilam when sending a PR, set the description as (resolves Issuenumber) to automatically close the issue after the merge.

File: cmd/create.go [hunk 1]
```diff
@@ -29,7 +30,6 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/mholt/archiver"
```
Comment (ChanakaCooray) [inline cmd/create.go]: Check the possibility of removing this dependency from glide.

**Comment by nishadi** [comment]
Shall we state the goals in PR template?

File: cmd/create.go [hunk 2]
```diff
@@ -1085,3 +1085,67 @@ func copyFile(filename string, locationInUpdate, relativeLocationInTemp string,
...
+
+		file, err := os.Open(path)
+		if err != nil {
+			return err
+		}
+		defer file.Close()
+		_, err = io.Copy(writer, file)
+		return err
+	})
+
```
Comment (nishadi) [inline cmd/create.go]: Shall we remove the additional spaces?

File: cmd/create.go (hunk 1 above)
Comment (irshadnilam) [reply cmd/create.go]: Yes. It is possible. I removed the dependency from glide.yaml

**Pull request merged**